src/api/moviesApi.ts

```diff
@@ -0,0 +1,8 @@
+export const moviesApis = {
+    GetMovies: (movieGenre?: string, sortBy = 'release_date') => {
```

Comment:
By convention PascalCase is used only for class names, other type of data better name with camelCase

Reply:
changed